packages/common/src/types.ts

```diff
@@ -0,0 +1,2 @@
+/** Hex-encoded string (without a 0x prefix) */
+export type Hex = string; // todo: should prefix always be allowed?
```

Comment:
The everlasting question

Reply:
lol -- I'm starting to consider it more and more, one more if shouldn't hurt perf 😅